crates/services/p2p/src/gossipsub/topics.rs

```diff
@@ -9,15 +9,14 @@ use super::messages::{
     GossipsubBroadcastRequest,
 };
 
-pub type GossipTopic = Sha256Topic;
 pub const NEW_TX_GOSSIP_TOPIC: &str = "new_tx";
 
 /// Holds used Gossipsub Topics
-/// Each field contains TopicHash and GossipTopic itself
-/// in order to avoid converting GossipTopic to TopicHash on each received message
+/// Each field contains TopicHash and Sha256Topic itself
+/// in order to avoid converting Sha256Topic to TopicHash on each received message
 #[derive(Debug)]
 pub struct GossipsubTopics {
-    new_tx_topic: (TopicHash, GossipTopic),
+    new_tx_topic: (TopicHash, Sha256Topic),
```

Comment:
```suggestion
    new_tx_topic: TopicHash,
```

I don't see any reason to store it, it is not used=D

Reply:
removed